data/model/namespacequota.py

```diff
@@ -33,6 +33,8 @@ def verify_namespace_quota(repository_ref):
 def verify_namespace_quota_force_cache(repository_ref):
     force_cache_repo_size(repository_ref)
     namespace_size = get_namespace_size(repository_ref.namespace_name)
+    if namespace_size is None:
+        namespace_size = 0
```

Comment:
if this every happens throw error dont send 0 it will result in a non-functional hard to debug error.

Reply:
That one is correct however the one above it is incorrect in its design. That one is not assuming that the previous line has created the sizing and does have the option of valid 0.